lib/lsp-data/oit_person_to_alma.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+module LspData
+  # convert a OIT person hash into alma xml
+  class OitPersonToAlma
+    attr_reader :person, :xml
+
+    # @param xml [XMLBuilder] builder to insert the person into
+    # @param person [Hash] oit hash of a person
+    def initialize(person:, xml:)
+      @person = person
+      @xml = xml
+    end
+
+    def convert
```

Comment:
For me, there is a bit of a mismatch between the name `convert` and this method's behavior.  This method both converts the data and also adds it to the `xml` instance variable.  If you call `convert` multiple times, for example, it adds duplicates of the same data to the xml object.  I wonder if a name like `add_to_xml` would make the behavior clearer?  Alternatively, maybe the behavior of this method could be just to convert the data, rather than converting and mutating the xml?

Reply:
I think that this is pointing to a good refactoring opportunity, to separate out the conversion of the data from applying to the XML. I will attempt this today. Thank you for the pointer!

Reply:
I got rubocop to not complain, but do you think readability was sacrificed?